onyo/lib/items.py

```diff
@@ -319,27 +319,34 @@ def get_path_name(self) -> Path | None:
 
     def is_asset(self) -> bool | None:
         """Initializer for the ``'onyo.is.asset'`` pseudo-key."""
-
```

Comment:
Nit: (and it's really not important), but I've been adding this blank space.

And once upon a time linting tools agreed with me, but now they disagree (due to PEP-something). So there's no spaces for functions but there are spaces for everything else.

If your opinion is pro PEP-something, then that's 2 votes against my 1, and I can no longer add them (and enable that `ruff` rule so that we don't drift).

Reply:
I don't really care. If you like it, I'll add it in again. :)

Reply:
I don't have a strong opinion. I was kinda hoping you did. :laughing: 

It doesn't matter. Ignore this and it's a lint for another day.